ios/Classes/FlutterArkitView+Handlers.swift

```diff
@@ -245,6 +245,24 @@ extension FlutterArkitView {
       result(nil)
     }
   }
+    func onCameraIntrinsics(_ result: FlutterResult) {
```

Comment:
Please reformat this file with [swiftformat](https://github.com/nicklockwood/SwiftFormat).